### PR TITLE
Add missing types to Transaction in index.d.ts

### PR DESCRIPTION
**Pull request by nojvek**
payment_channel and authorized_date are missing. Typescript is yelling at me even though I see it being returned from the api.

### CODE REVIEW

**Comment by nojvek** [comment]
fyi @otherchen ^

**Comment by nojvek** [comment]
Kind ping @skylarmb ^

File: index.d.ts [hunk 1]
```diff
@@ -308,8 +308,10 @@ declare module 'plaid' {
     category: Array<string> | null;
     category_id: string | null;
     date: Iso8601DateString;
+    authorized_date: Iso8601DateString;
```
Comment (skylarmb) [inline index.d.ts:311]: hey there, this field is nullable. mind updating? https://plaid.com/docs/#transaction-schema

Comment (nojvek) [reply index.d.ts]: done

**Comment by skylarmb** [comment]
@nojvek thanks for this update! it should be released in the next version of plaid-node on Thursday 4/14

**Pull request merged**